gradle/lib/dependabot/gradle/requirement.rb

```diff
@@ -36,7 +36,7 @@ def initialize(*requirements)
           # NOTE: Support ruby-style version requirements that are created from
           # PR ignore conditions
           version_reqs = req_string.split(",").map(&:strip)
-          if version_reqs.all? { |s| Gem::Requirement::PATTERN.match?(s) }
+          if req_string.include?(",") && version_reqs.all? { |s| PATTERN.match?(s) }
```

Comment:
I missed that `PATTERN` includes the Gem::Requirement::OPS` and the `PATTERN` from `Gradle::Version` so does exactly what we want here.